src/models/Order.js

```diff
@@ -84,11 +91,16 @@ const updateOrderStatus = async () => {
   }
 };
 
-// Call the updateOrderStatus function every 1 minute (60 * 1000 milliseconds)
-const intervalId = setInterval(updateOrderStatus, 1 * 60 * 1000);
+// Call the updateOrderStatus function every .... x time
+
+//const intervalId = setInterval(updateOrderStatus, process.env.PROD_TIME_DURATION); //prod 1h
+const intervalId = setInterval(
+  updateOrderStatus,
+  process.env.DEV_TIME_DURATION
```

Comment:
This should also vary by prod vs dev right? 

```suggestion
  timeDuration // in prod 1hr, in dev 20hrs
```

Reply:
@akosasante will commit it, please check if I did it correctly- I had to move  ```
let isDevelopment = process.env.NODE_ENV !== 'production';
    const timeDuration = isDevelopment
      ? parseInt(process.env.DEV_ORDER_EXPIRY_DURATION)
      : parseInt(process.env.PROD_ORDER_EXPIRY_DURATION); 
```to the globe scope so it could be accesible both from **updateOrderStatus** and i**ntervalId**